### PR TITLE
feat: AE coin markets

### LINKED ISSUE #822: AE coin markets
**Acceptance criteria:**
- A new tab is visibile in the AE coin page called "Markets"
- Inside the tab a table is present with the following columns:
  - Exchange name
  - Token pair
  - Price
  - Volume (24h)
- Data is cached, some endpoints cannot be filtered
- Price and Volume are expressed in $

API Endpoints:
- Gate.io API: https://api.gateio.ws/api/v4/spot/tickers?currency_pair=AE_USDT
- Mexc: https://api.mexc.com/api/v3/ticker/24hr?symbol=AEUSDT
- Coinstore: https://api.coinstore.com/api/v1/market/tickers
- Hotcoin: https://api.hotcoinfin.com/v1/market/ticker
- CoinW API: https://api.coinw.com/api/v1/public?command=returnTicker

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #822 

## Demo

https://github.com/aeternity/aescan/assets/15363559/8e63db15-2f31-4573-9096-51042c4b4940

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -9,6 +9,13 @@ export default defineNuxtConfig({
   devServer: {
     port: 8080,
   },
+  nitro: {
+    routeRules: {
+      '/proxy/gate': { proxy: 'https://api.gateio.ws/api/v4/spot/tickers?currency_pair=AE_USDT', cors: true },
```
Comment (janmichek) [inline nuxt.config.ts:14]: Some of the request needed to be passed through proxy to get over CORS. Some links didn't need this approach

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
@janmichek  please check the description for 3 first column: 


https://github.com/aeternity/aescan/assets/69896204/4d02fb74-948c-465b-a65b-e3ae2dca2224

**Comment by janmichek** [comment]
> @janmichek please check the description for 3 first column:
> 2024-06-13.1.33.59.mov

Thanks, fixed

**Comment by Liubov-crypto** [comment]
LGTM

File: src/components/AeCoinMarketsTableCondensed.vue [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        </tr>
+        <tr class="ae-coin-markets-table-condensed__row">
+          <th class="ae-coin-markets-table-condensed__header">
+            Price
+            <hint-tooltip>
+              {{ aeCoinHints.marketPrice }}
+            </hint-tooltip>
+          </th>
+          <td class="ae-coin-markets-table-condensed__data">
+            $ {{ formatNullable(formatNumber(coinStore?.coinW)) }}
```
Comment (michele-franchi) [inline src/components/AeCoinMarketsTableCondensed.vue:237]:
```suggestion
            $ {{ formatNullable(formatNumber(coinW?.price)) }}
```

File: src/components/AeCoinMarketsTableCondensed.vue [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+        </tr>
+        <tr class="ae-coin-markets-table-condensed__row">
+          <th class="ae-coin-markets-table-condensed__header">
+            Volume (24h)
+            <hint-tooltip>
+              {{ aeCoinHints.volume }}
+            </hint-tooltip>
+          </th>
+          <td class="ae-coin-markets-table-condensed__data">
+            $ {{ formatNullable(formatNumber(coinStore?.coinW)) }}
```
Comment (michele-franchi) [inline src/components/AeCoinMarketsTableCondensed.vue:248]:
```suggestion
            $ {{ formatNullable(formatNumber(coinW?.volume)) }}
```

**Review by michele-franchi — CHANGES_REQUESTED** [review]

File: src/components/AeCoinMarketsTableCondensed.vue (hunk 2 above)
Comment (janmichek) [reply src/components/AeCoinMarketsTableCondensed.vue]: fixed

**Pull request merged**